packages/cli/src/daemon-config.ts

```diff
@@ -347,6 +347,12 @@ export class DaemonMetricsConfig {
 @jsonObject
 @toJson
 export class DaemonConfig {
+  /**
+   * An array of DIDs with access to Admin API (represented as strings)
+   */
+  @jsonArrayMember(String, { name: 'admin-dids' })
```

Comment:
seems like this should probably go into a sub-section of the config file rather than the top level.  I'd suggest putting it under `node` (ie in the `DaemonCeramicNodeConfig`)